Publish stage is failing

The most recent merge to the default branch failed:

```
Run make test-release
  make test-release
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.9.18/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.9.18/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.9.18/x64
    Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.9.18/x64
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.9.18/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.9.18/x64/lib
    TWINE_USERNAME: __token__
    TWINE_PASSWORD: ***
    TWINE_REPOSITORY_URL: https://test.pypi.org/legacy/
    TWINE_NON_INTERACTIVE: true
/opt/hostedtoolcache/Python/3.9.18/x64/bin/python: No module named setuptools_scm
python -m build
/opt/hostedtoolcache/Python/3.9.18/x64/bin/python: No module named build
make: *** [Makefile:61: dist/awscli_login--py3-none-any.whl] Error 1
Error: Process completed with exit code 2.
```
https://github.com/techservicesillinois/awscli-login/actions/runs/8300830074/job/22719747525

--- a/.github/workflows/cicd.yml
+++ b/.github/workflows/cicd.yml
@@ -201,10 +201,14 @@ jobs:
       with:
         name: build
 
+    # setuptools_scm sometimes will regenerate _version.py which causes
+    # Makefile to rebuild things. To avoid unnecessary rebuilds we set the
+    # following environment variable.
     - name: Set setuptools_scm environment variable
       shell: bash
       run: |
-        echo SETUPTOOLS_SCM_PRETEND_VERSION_FOR_AWSCLI_LOGIN=`PYTHONPATH=src python -c 'from awscli_login._version import __version__; print(__version__)'` >> $GITHUB_ENV
+        PYTHONPATH=src python -c 'from awscli_login._version import __version__; print(f"SETUPTOOLS_SCM_PRETEND_VERSION_FOR_AWSCLI_LOGIN={__version__}")' > "$RUNNER_TEMP/_awscli_version"
+        cat "$RUNNER_TEMP/_awscli_version" >> "$GITHUB_ENV"
 
     - name: Touch build artifacts
       run: |
@@ -277,6 +281,15 @@ jobs:
       uses: actions/download-artifact@v3
       with:
         name: build
+
+    # setuptools_scm sometimes will regenerate _version.py which causes
+    # Makefile to rebuild things. To avoid unnecessary rebuilds we set the
+    # following environment variable.
+    - name: Set setuptools_scm environment variable
+      shell: bash
+      run: |
+        PYTHONPATH=src python -c 'from awscli_login._version import __version__; print(f"SETUPTOOLS_SCM_PRETEND_VERSION_FOR_AWSCLI_LOGIN={__version__}")' > "$RUNNER_TEMP/_awscli_version"
+        cat "$RUNNER_TEMP/_awscli_version" >> "$GITHUB_ENV"
 
     - name: Publish to PyPI
       if: startsWith(github.ref, 'refs/tags')

